core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -580,7 +580,13 @@ private[deploy] class Master(
    * The number of cores assigned to each executor is configurable. When this is explicitly set,
    * multiple executors from the same application may be launched on the same worker if the worker
    * has enough cores and memory. Otherwise, each executor grabs all the cores available on the
-   * worker by default, in which case only one executor may be launched on each worker.
+   * worker by default, in which case only one executor may be launched on each worker during one
```

Comment:
Shouldn't this line be:
worker by default, in which case only one executor per application may be launched on each worker during one